tests/rptest/clients/rpk.py

```diff
@@ -128,11 +128,11 @@ def alter_topic_config(self, topic, set_key, set_value):
         cmd = ['alter-config', topic, "--set", f"{set_key}={set_value}"]
         self._run_topic(cmd)
 
-    def wasm_deploy(self, script, name, description):
+    def wasm_deploy(self, script, name, description, wasm_type="async"):
```

Comment:
`wasm_type` can be an enum, enforcing the rule that there can only be two possible values to pass to this method. Accepting an arbitrary string has the type system infer to users that any value can be passed, which is not true.